src/GitHub.VisualStudio/Views/Dialog/GitHubDialogWindow.xaml

```diff
@@ -25,9 +24,9 @@
                    WindowStartupLocation="CenterOwner"
                    Content="{Binding Content}">
   <Window.Resources>
-    <views:ViewLocator x:Key="viewLocator"/>
-    <DataTemplate DataType="{x:Type vm:ViewModelBase}">
+    <!--<ghfvs:ViewLocator x:Key="viewLocator"/>-->
```

Comment:
Why is this now commented out?

Reply:
Oops, oversight! That shouldn't be commented out (and should use `local:`).